.github/workflows/test.yml

```
@@ -0,0 +1,51 @@
+name: Run unit and style tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - master
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Run tests on PHP ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental == true }}
+
+    strategy:
+      matrix:
+        php:
+          - '5.4'
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          ini-values: error_reporting=E_ALL, display_errors=On
+          coverage: none
+
+      - name: Install Composer dependencies
+        uses: ramsey/composer-install@v1
+
+      - name: 'Linting code'
+        run: ./vendor/bin/parallel-lint --exclude vendor .
```

```suggestion
        run: ./vendor/bin/parallel-lint --exclude vendor --exclude git .
```

(generally makes the run slightly faster as the `git` directory can be large)

---

You may also want to consider adding the `cs2pr` tool in the `setup-php` step and then adding `--checkstyle | cs2pr` to this command to allow results to be displayed inline in PRs.